veto/vetoHit.cxx

```diff
@@ -1,106 +1,116 @@
+    nav->LocalToMaster(origin, master);
+    TVector3 pos = TVector3(master[0], master[1], master[2]);
     return pos;
 }
 Double_t vetoHit::GetX()
-{ TVector3 pos = GetXYZ();
-  return pos.X();
+{
+    TVector3 pos = GetXYZ();
+    return pos.X();
 }
 Double_t vetoHit::GetY()
-{ TVector3 pos = GetXYZ();
-  return pos.Y();
+{
+    TVector3 pos = GetXYZ();
+    return pos.Y();
 }
 Double_t vetoHit::GetZ()
-{ TVector3 pos = GetXYZ();
-  return pos.Z();
+{
+    TVector3 pos = GetXYZ();
+    return pos.Z();
 }
 TGeoNode* vetoHit::GetNode()
 {
-   TGeoNode* node;
-   TGeoNavigator* nav = gGeoManager->GetCurrentNavigator();
-   TString path = "cave/DecayVolume_1/T2_1/VetoLiSc_0/";
-   //id = ShapeType*100000 + blockNr*10000 + Zlayer*100 + number*10 + position;
-   Int_t ShapeType = fDetectorID/100000;
-   Int_t blockNr = (fDetectorID-ShapeType*100000)/10000;
-   Int_t Zlayer = (fDetectorID-ShapeType*100000-blockNr*10000)/100;
-   Int_t number = (fDetectorID-ShapeType*100000-blockNr*10000-Zlayer*100)/10;
+    TGeoNode* node;
+    TGeoNavigator* nav = gGeoManager->GetCurrentNavigator();
+    TString path = "cave/DecayVolume_1/T2_1/VetoLiSc_0/";
+    // id = ShapeType*100000 + blockNr*10000 + Zlayer*100 + number*10 + position;
+    
+    Int_t ShapeType   = fDetectorID / 100000;
+    Int_t blockNr     = (fDetectorID % 100000) / 10000;
+    Int_t Zlayer      = (fDetectorID % 10000) / 100;
+    Int_t number      = (fDetectorID % 100) / 10;
+    //Int_t position    = (fDetectorID % 10);
 
-   if(ShapeType==1)path+="LiScX_";
-   else if(ShapeType==2)path+="LiScY_";
-   else if(ShapeType==3)path+="LiSc_L1_";
-   else if(ShapeType==4)path+="LiSc_R1_";
-   else if(ShapeType==5)path+="LiSc_L2_";
-   else if(ShapeType==6)path+="LiSc_R2_";
+    if (ShapeType == 1)
+        path += "LiScX_";
+    else if (ShapeType == 2)
+        path += "LiScY_";
+    else if (ShapeType == 3)
+        path += "LiSc_S3_";
+    else if (ShapeType == 4)
+        path += "LiSc_S4_";
+    else if (ShapeType == 5)
+        path += "LiSc_S5_";
+    else if (ShapeType == 6)
+        path += "LiSc_S6_";
 
-   path+=ShapeType;
-   path+=blockNr;
-   if(ShapeType<3){
-        if(Zlayer<10)path+="0";
-        path+=Zlayer;
-   }
-   else {
-        path+="00";
-   }
-   path+=number;
-   path+="0_";
-   path+=fDetectorID;
+    path += ShapeType;
+    path += blockNr;
+    if (Zlayer < 10)
+        path += "0";
+    path += Zlayer;
+    path += number;
+    path += "0_";
+    path += fDetectorID;
 
-   nav->cd(path);
-   node=nav->GetCurrentNode();
-   return node;
+    nav->cd(path);
+    node = nav->GetCurrentNode();
+    return node;
 }
 
-
-
 // -----   Public method Print   -------------------------------------------
 void vetoHit::Print(Int_t detID) const
 {
-  cout << "-I- vetoHit: veto hit " << " in detector " << fDetectorID << endl;
-  cout << "  ADC " << fdigi << " ns" << endl;
+    LOG(DEBUG) << "-I- vetoHit: veto hit " << " in detector " << fDetectorID ;
```

Comment:
I would remove the `-I-` here, as it indicates the INFO severity, and make the severity for the LOG(INFO) to match the past behaviour. DEBUG isn't printed by default.